src/libraries/Microsoft.PowerFx.Core/Localization/Strings.cs

```diff
@@ -97,7 +101,12 @@ internal static class TexlStrings
         public static StringGetter AboutBooleanN = (b) => StringResources.Get("AboutBooleanN", b);
         public static StringGetter BooleanNArg1 = (b) => StringResources.Get("BooleanNArg1", b);
         public static StringGetter AboutBooleanNT = (b) => StringResources.Get("AboutBooleanNT", b);
-        public static StringGetter BooleanNTArg1 = (b) => StringResources.Get("BooleanNTArg1", b);
+        public static StringGetter BooleanNTArg1 = (b) => StringResources.Get("BooleanNTArg1", b);
+
+        public static StringGetter AboutBooleanW = (b) => StringResources.Get("AboutBooleanW", b);
+        public static StringGetter BooleanWArg1 = (b) => StringResources.Get("BooleanWArg1", b);
+        public static StringGetter AboutBooleanWT = (b) => StringResources.Get("AboutBooleanWT", b);
+        public static StringGetter BooleanWTArg1 = (b) => StringResources.Get("BooleanWTArg1", b);
 
```

Comment:
Does this function need it's own strings? Is there a reason it shouldn't reuse Boolean's?

I see it was done for BooleanN and BooleanB, but that seems kind of silly as well.